runtime/doc/api.txt

```diff
@@ -2162,7 +2162,9 @@ nvim_buf_delete({buffer}, {opts})                          *nvim_buf_delete()*
       • {buffer}  Buffer handle, or 0 for current buffer
       • {opts}    Optional parameters. Keys:
                   • force: Force deletion and ignore unsaved changes.
-                  • unload: Unloaded only, do not delete. See |:bunload|
+                  • unload: Unload only, do not delete. See |:bunload|
+                  • delete: Delete buffer only, do not wipe it out. See
```

Comment:
We should really try to avoid mutually-exclusive options, they are hard to explain and hard to use. 

When we add boolean options (such as `unload` in the above case) we should ask "is a boolean actually enough or should it be a (enum-like) string option", to avoid this situation where we have multiple mutually-exclusive booleans.